scripts/commands/start.py

```diff
@@ -7,7 +7,11 @@
 import os
 
 from .. import filesystem
-from .command import Command
+from .command import (
+    Command,
+    FAILURE,
+    SUCCESS,
```

Comment:
I think the two imports above are in the wrong place - we should import from `..` instead.